### PR TITLE
Basic frame for zuul unit tests (#12)

### PR DESCRIPTION
**Pull request by dietmax**
I wrote a basic frame as a basis to start implementing more zuul unit tests with the example of the trivial never() and always() functions. The unit tests can be run by executing "zuul --local 8080 --ui mocha-qunit -- trivialtest.js" after navigating to the /tests/ directory

If this is continued, we would have to think of a solution to test the functions within the dragula()-function scope though, as those can't be accessed inside the test as of now.

### CODE REVIEW

**Comment by bevacqua** [comment]
Looks great but let's not break the API over this. We could just write a test that asserts that `typeof dragula === 'function'`.

**Comment by dietmax** [comment]
Alright, I restored the API as it was before and did just that

File: tests/trivialtest.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var assert = require('assert');
+
+// our test file needs to know where to find the module relative to itself
+var dragula = require('../dragula');
+
+// general category for the tests that follow
+suite('trivial');
```
Comment (bevacqua) [inline tests/trivialtest.js]: What is `suite`?

File: tests/trivialtest.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+test('dragula should be a function', function() {
```
Comment (bevacqua) [inline tests/trivialtest.js]: What is `test`?

**Comment by bevacqua** [comment]
I think we'd be much better off using `testron` with `tape`

**Comment by dietmax** [comment]
After rewriting I agree, now it's much cleaner and easier to use.

**Comment by bevacqua** [comment]
Sweetness. Please squash your commits and we can merge this in.

If you don't know how to squash commits:

```
git reset HEAD~4
git add .
git commit -am "setting up test harness with tape and testron"
git push --force
```

**Comment by dietmax** [comment]
Thanks, I did not know that indeed, ready for merge.

**Pull request merged**